qa-pipelines/set-pipeline

```diff
@@ -75,12 +88,18 @@ fly \
         # refer to secret vars correctly
         ${secrets_file:+gpg --decrypt --batch ${secrets_file}} # Import secrets
         sed '/^---$/d' < "${vars_file}"                        # Config vars
+        sed '/^---$/d' < "${flags_file}"
+        ruby -r yaml << EOF | tail -n+2
+          pipeline_flags=YAML.load_file('${flags_file}').keys
+          all_flags=YAML.load_file('flags.yml').keys
```

Comment:
I still think the default should be "false" in flags.yml

Reply:
I don't see as much value in maintaining a separate file with all flags set to false, since we would never use it as the preset.